### DISCUSSION TITLE
How to opt back in to `focus` in `refetchOnWindowFocus`?

### DISCUSSION THREAD

**Discussion by italodeandra** [TanStack/query#8023] [Q&A] [answered]
Since this was fixed https://github.com/TanStack/query/pull/4805 I'm having issue to keep some queries always update if the user goes to a second screen and gets back to the app window. I understand the reason behind it but I wish it was easier to opt back to use `focus` instead of just visibility change.

Is there an easy way to use the same implementation as it was before? Instead of relying on workarounds?

**Comment by TkDodo** [comment, marked as answer]
yes, you can register your own event listener: https://tanstack.com/query/latest/docs/reference/focusManager#focusmanagerseteventlistener
